res/database/Default/database_scripts.xml

```diff
@@ -610,30 +619,35 @@
 			<variables>
 				<title>
 					<de>%ANIMAL1%%CONNECTOR1%|Party mit %ANIMAL1%%CONNECTOR1%|Zu Besuch bei %ANIMAL1%%CONNECTOR2%|Auf der Spur von %ANIMAL1%%CONNECTOR2%|Im Zoo bei %ANIMAL1% und %ANIMAL2%</de>
+					<en>%ANIMAL1%%CONNECTOR1%|Party with %ANIMAL1%%CONNECTOR1%|Visiting %ANIMAL1%%CONNECTOR2%|On the trail of %ANIMAL1%%CONNECTOR2%|In the Zoo among %ANIMAL1% and %ANIMAL2%</en>
 				</title>
 				<animals>
 					<de>%ANIMAL1%, %ANIMAL2% und viele weitere Tiere|Tiere im Zoo.|Pfleger, Gäste und natürlich Tiere im Gehege.</de>
+					<en>%ANIMAL1%, %ANIMAL2% and many more animals|animals in the zoo.|keepers, guests and of course animals in the enclosure.</en>
 				</animals>
 				<extra>
 					<de>%EXTRA1%%EXTRA2%.||Tierpfleger geben Tipps für die Aufzucht zuhause.|Inklusive Tipps für den Umgang mit Haustieren.</de>
+					<en>%EXTRA1%%EXTRA2%.||Animal keepers give tips for rearing at home.|Including tips for dealing with pets.</en>
 				</extra>
 				<extra1>
-					<de>Niedliche Tierbabies |Knuddelige Wildkatzen |Bio-Dung</de>
+					<de>Niedliche Tierbabies |Knuddelige Wildkatzen |Bio-Dung </de>
+					<en>Cute baby animals |Cuddlesome feral cats |Bio manure </en>
 				</extra1>
 				<extra2>
-					<de>zu hauf.|sowie ab und zu eine finale Spritze| und ab und zu eine Tierfütterung aus Eigenanbau</de>
+					<de>zu hauf.|sowie ab und zu eine finale Spritze|und ab und zu eine Tierfütterung aus Eigenanbau</de>
+					<en>in abundance.|and every now and then a final injection|and from time to time animal feeding from cultivation grown on-location</en>
```

Comment:
fodder grown on-site?

Reply:
feeding from fodder hmmm...

but on-site is fine, I'm gonna take that one.